elasticsearch/src/main/java/org/hibernate/search/elasticsearch/impl/ElasticsearchIndexWorkVisitor.java

```diff
@@ -101,9 +99,17 @@ public ElasticsearchIndexWorkVisitor(String indexName, boolean refreshAfterWrite
 
 	@Override
 	public BackendRequest<?> visitOptimizeWork(OptimizeLuceneWork work, Void p) {
-		// TODO HSEARCH-2092 implement
-		LOG.warn( "Optimize work is not yet supported for Elasticsearch, ignoring it" );
-		return null;
+		/*
+		 * As of ES 2.1, the Optimize API has been renamed to ForceMerge,
+		 * but Jest still does not provide commands for the ForceMerge API as of
+		 * version 2.0.3
+		 * See https://github.com/searchbox-io/Jest/issues/292
```

Comment:
Currently no, Jest really doesn't even know about the newer API.

But to Elasticsearch, it seems both APIs are the same: [the docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-optimize.html) say the Optimize API has been "renamed", and the descriptions are really close (almost copy/paste).